.github/workflows/lint-changed.yml

```
@@ -17,19 +17,41 @@ jobs:
     if: ${{ github.actor != 'OSBotify' || github.event_name == 'workflow_call' }}
     runs-on: ubuntu-latest
     steps:
+      - name: Fetch merge base SHA from API
+        id: merge_base
+        run: |
+          MERGE_BASE_SHA="$(gh api repos/"$REPO"/compare/"$PR_BASE_SHA"..."$PR_HEAD_SHA" \
```

I find it more readable in bash to quote the entire strings, though quoting individual variables like you did is valid

```suggestion
          MERGE_BASE_SHA="$(gh api "repos/$REPO/compare/${PR_BASE_SHA}...${PR_HEAD_SHA}" \
```